backend/src/user/structures/Provider.ts

```diff
@@ -1,5 +1,5 @@
 import { Union, Literal, Static } from "runtypes";
 
-export const Provider = Union(Literal("google"));
+export const Provider = Union(Literal("google"), Literal("algodebug"));
```

Comment:
Nie wprowadzajmy takiego providera, bo to może być mylące. Te aktualne projekty, które są jako publiczne traktujmy jakby były wprowadzone przez jakiegoś istniejącego użytkownika (który docelowo będzie adminem, ale to już inny temat). I w tym polu authorId niech wskazują na tego użytkownika.

Reply:
Tylko wtedy jakiego użytkownika ustawić? Jeśli provider miałby być google to wtedy co mam dać w id?

Reply:
W sumie nie ma to teraz większego znaczenia, może być jakikolwiek użytkownik. Proponuję tego użytkownika z adresem algodebug.kni@gmail.com (jest na produkcyjnej bazie ten użytkownik zapisany w kolekcji users).